src/SimpleExecutionController.cpp

```diff
@@ -186,15 +191,15 @@ int SimpleExecutionController::main() {
             continue;
         }
 
-        if (this->abort || this->num_completed_jobs == this->workload_spec.size()) {
+        if (this->abort || this->workload_spec.size() == 0) {
             break;
         }
     }
 
     wrench::Simulation::sleep(10);
 
     WRENCH_INFO("--------------------------------------------------------")
-    if (this->num_completed_jobs == this->workload_spec.size()){
+    if (this->workload_spec.size() == 0){
```

Comment:
Is this an equivalent change?

Reply:
It is kind of a bug fix, which is not necessarily part of this PR. It implements the check, as it was supposed to be.